lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -511,7 +511,11 @@ WebMapServiceCatalogItem.prototype.updateFromCapabilities = function(capabilitie
     updateValue(this, overwrite, 'minScaleDenominator', thisLayer.MinScaleDenominator);
     updateValue(this, overwrite, 'getFeatureInfoFormats', getFeatureInfoFormats(capabilities));
     updateValue(this, overwrite, 'rectangle', getRectangleFromLayers(this._allLayersInRawMetadata));
-    updateValue(this, overwrite, 'intervals', getIntervalsFromLayer(this, thisLayer));
+    this._lazyLoadInterval = function() {
```

Comment:
Maybe call it `_lazyLoadIntervals` instead? And make sure it's declared as undefined in the constructor :).